frontend/components/base/render-markdown/RenderMarkdown.base.component.vue

```diff
@@ -24,9 +24,11 @@ export default {
     },
   },
   created() {
-    const parsed = marked.parse(this.markdown, {
+    const cleanMarkdown = this.markdown.replace(/[^\S\r\n]+$/gm, "");
```

Comment:
externalized the logic to cleanmarkdown in a method (instead of a computed since it's called only once) and add some unit test to it